operations/last_revision.go

```diff
@@ -130,15 +141,21 @@ type lastRevisionCriteria struct {
 	// minSuccessProportion is a criterion for the minimum proportion of tasks
 	// in a matching build that must succeed.
 	minSuccessProportion float64
+	// successfulTasks is a criterion for the list of task names that must have
+	// succeeded in the build.
```

Comment:
The flag name says "names of tasks that, if present in the builds, must have succeeded" and the implementation seems to align with that. This is implying that they must be there and they must be successful. Can this be reworded?

Reply:
Reworded a little bit to match the CLI description.